envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt

```diff
@@ -13,10 +14,7 @@ class EnvoyContainer(
     private val envoyControl1XdsPort: Int,
     private val envoyControl2XdsPort: Int = envoyControl1XdsPort
 ) : GenericContainer<EnvoyContainer>(ImageFromDockerfile().withDockerfileFromBuilder {
-    it.from("envoyproxy/envoy-alpine:latest") // TODO (https://github.com/allegro/envoy-control/issues/7): NOT latest,
-            // whatever is tagged latest in local cache is considered latest, C'MON
-            // this should be possible to overcome soon: https://github.com/moby/moby/issues/13331
-            // but it's not great in the long run if future updates break the build
+    it.from("envoyproxy/envoy-alpine-dev:b7bef67c256090919a4585a1a06c42f15d640a09")
```

Comment:
We should explain why we are using this version. Maybe add link to issue/pr with fix in envoy.

Reply:
Done. I also set the correct envoy version in tools/ Dockerfile.